app/configuration.rb

```diff
@@ -27,8 +27,9 @@ module GitlabMonitor
     #Implementation of class responsible for showing notifications in system. Specify hide timeout [s].
     #You can use equations like 60*5 -> 5 minutes
     config.notifier          = (RbConfig::CONFIG['host_os'].match(/mswin|windows/i) \
-                                  ? WindowsNotificationExecutor 
+                                  ? WindowsNotificationExecutor
                                   : LinuxNotificationExecutor).new(time: 5)
+    config.link_enabled        = true
```

Comment:
Ale to drugie żeby działało i tak potrzebuje pierwszego, a to jest właśnie brzydkie i psuje czytelność konfiguracji, czyli to czego chciałem uniknąć. Swoją drogą to ten wybór wykonawcy powiadomień powinien być już wyciągnięty z tej klasy bo to już nie jest konfiguracja skoro jest automatycznie wybierany, a wtedy tym bardziej ten konfig do linków musi zostać tam gdzie jest. Prawda, że chciałem uniknąć też czytania konfiguracji bezpośrednio przez wykonawcę i mieć to w konstruktorze, co by się dało zrobić uprzednio wyrzucając z konfiga decyzję o notyfikatorze.